toxcore/tox.h

```diff
@@ -2234,15 +2252,16 @@ const char *tox_err_file_send_to_string(Tox_Err_File_Send value);
  * - If the file size was increased
  *   - and sending mode was streaming (file_size = UINT64_MAX), the behaviour
  *     will be as expected.
- *   - and sending mode was file (file_size != UINT64_MAX), the file_chunk_request
- *     callback will receive length = 0 when Core thinks the file transfer has
- *     finished. If the client remembers the file size as it was when sending the
- *     request, it will terminate the transfer normally. If the client re-reads the
- *     size, it will think the friend cancelled the transfer.
+ *   - and sending mode was file (file_size != UINT64_MAX), the
+ *     file_chunk_request callback will receive length = 0 when Core thinks the
```

Comment:
Maybe "Tox" instead of "Core"?

Reply:
Some parts of the API refer to the library as "Tox" and "Tox library", other as "toxcore" and "the toxcore library", other yet as "Core"...

Reply:
I have tried addressing this in a separate commit, please check.

Reply:
Oh, I actually missed this particular instance of "Core". Let me update it.

Reply:
There you go :)